core/src/test_utils.rs

```diff
@@ -82,6 +82,7 @@ macro_rules! create_regtest_rpc {
                     self.2.display()
                 );
                 let _ = self.0.kill();
+                let _ = self.rpc(); // To get away from "unused" warnings
```

Comment:
can we add docs that says this macro modifies the config? maybe even rename to reflect that